src/WinRPM.jl

```diff
@@ -459,25 +465,18 @@ function do_install(package::Package)
         error("failed to download $name $(data[2]) from $source/$path.")
     end
     cache = getcachedir(source)
-    path2 = joinpath(cache,escape(path))
+    path2 = joinpath(cache, escape(path))
     open(path2, "w") do f
         write(f, data[1])
     end
     info("Extracting: ", name)
-
-    if VERSION < v"0.7.0-DEV.2181"
-        cpio = splitext(path2)[1]*".cpio"
-    else
-        cpio = splitext(joinpath(cache, escape(basename(path))))[1] * ".cpio"
-    end
-
+    cpio = splitext(path2)[1]*".cpio"
     local err = nothing
-    for cmd = [`$exe7z x -y $path2 -o$cache`, `$exe7z x -y $cpio -o$installdir`]
+    for cmd = (`$exe7z x -y $path2 -o$cache`, `$exe7z x -y $cpio -o$installdir`)
         (out, pc) = open(cmd, "r")
         stdoutstr = read(out, String)
         if !success(pc)
             wait_close(out)
-            println(stdoutstr)
```

Comment:
don't hide error output

Reply:
Ah, good catch - thought that was one of my debugging println